es_jsonschema/mapping.py

```diff
@@ -40,10 +46,48 @@ def __init__(self, *args, **kwargs):
         }
         # json string formats -> elasticsearch type
         self.__formats_map = {}
```

Comment:
be careful with using double underscore for properties (https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references) - they make it hard for subclassing ...